exporter/metric/metric.go

```diff
@@ -741,22 +745,60 @@ func expHistToDistribution[N int64 | float64](hist metricdata.ExponentialHistogr
 		Mean:          mean,
 		BucketCounts:  counts,
 		BucketOptions: bucketOptions,
-		Exemplars:     toDistributionExemplar[N](hist.Exemplars),
+		Exemplars:     toDistributionExemplar[N](hist.Exemplars, projectID),
 	}
 }
 
-func toDistributionExemplar[N int64 | float64](Exemplars []metricdata.Exemplar[N]) []*distribution.Distribution_Exemplar {
+func toDistributionExemplar[N int64 | float64](Exemplars []metricdata.Exemplar[N], projectID string) []*distribution.Distribution_Exemplar {
 	var exemplars []*distribution.Distribution_Exemplar
 	for _, e := range Exemplars {
-		// TODO: Add context []attachments. See https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TypedValue#exemplar
-		exemplars = append(exemplars, &distribution.Distribution_Exemplar{Value: float64(e.Value), Timestamp: timestamppb.New(e.Time)})
+		attachments := []*anypb.Any{}
+		if hasValidSpanContext(e) {
+			traceID, spanID := e.TraceID, e.SpanID
```

Comment:
nit looks like these local variables are only used once for copying. consider using them directly on L759